src/hdf5/cloudsat_netcdf_to_ioda.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2020-2024 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+#
+# contribution based on prototype by 2023, Isaac Moradi
+#
+
+import sys
+import netCDF4 as nc
+import numpy as np
+import datetime
+import time
+
+import pyiodaconv.ioda_conv_engines as iconv
+from pyiodaconv.orddicts import DefaultOrderedDict
+from pyiodaconv.def_jedi_utils import set_metadata_attributes, set_obspace_attributes
+from pyiodaconv.def_jedi_utils import epoch, iso8601_string
+import read_cloudsat
+
+float_missing_value = iconv.get_default_fill_val(np.float32)
+int_missing_value = iconv.get_default_fill_val(np.int32)
+long_missing_value = iconv.get_default_fill_val(np.int64)
+
+metaDataName = iconv.MetaDataName()
+obsValName = iconv.OvalName()
+
+# globals
+CLOUDSAT_WMO_sat_ID = 788
+
```

Comment:
add satellite height here too for now... these should be moved into general utility shortly